### PR TITLE
global: init structure

### PR DESCRIPTION
**Pull request by drjova**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,35 @@
+[![Build Status](https://travis-ci.org/inveniosoftware/invenio-search-js.svg?branch=master)](https://travis-ci.org/inveniosoftware/invenio-search-js)
```
Comment (jirikuncar) [inline README.md]: we usually use the `https://img.shields.io/<provider>/<user>/<repo>.svg`

File: example/package.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "invenio-search-js-demo",
+  "version": "0.0.1",
+  "description": "Invenio-Search-JS Demo",
+  "main": "",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Harris Tzovanakis <me@drjova.com> (http://drjova.com)",
```
Comment (jirikuncar) [inline example/package.json]: CERN + your name in AUTHORS?

File: gulpfile.js [hunk 3]
```diff
@@ -0,0 +1,176 @@
+var gulp = require('gulp');
```
Comment (jirikuncar) [inline gulpfile.js]: missing licence header?

File: package.json [hunk 4]
```diff
@@ -0,0 +1,63 @@
+{
+  "name": "invenio-search-js",
+  "version": "0.0.1",
+  "description": "Javascript search components for Invenio-Search",
+  "author": {
+    "name": "Harris Tzovanakis",
```
Comment (jirikuncar) [inline package.json]: CERN

File: package.json [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    "email": "me@drjova.com"
```
Comment (jirikuncar) [inline package.json]: info@invenio-software.org

**Comment by drjova** [comment]
:+1:

File: example/lib/invenio-search-js.js [hunk 6]
```diff
@@ -0,0 +1,303 @@
...
+        q: '',
+        index: '',
+        doc_type: '',
+      }
+    };
+
+    ////////////
+
+    /**
+    * @name invenioDoSearch
```
Comment (jirikuncar) [inline example/lib/invenio-search-js.js]: @drjova can you check the indentation here (see https://github.com/inveniosoftware/invenio-search-js/pull/1/files#diff-fc0759b7396267f05828baca8f6f544cR26)

File: gulpfile.js [hunk 7]
```diff
@@ -0,0 +1,198 @@
...
+ *
+ * Invenio is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Invenio; if not, write to the Free Software Foundation, Inc.,
+ * 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+* In applying this license, CERN does not
```
Comment (jirikuncar) [inline gulpfile.js]: ` ` ?

File: karma-dist-minified.conf.js [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+ *
+ * Invenio is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Invenio; if not, write to the Free Software Foundation, Inc.,
+ * 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+* In applying this license, CERN does not
```
Comment (jirikuncar) [inline karma-dist-minified.conf.js]: ` ` ?

File: karma-dist-concatenated.conf.js [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+ *
+ * Invenio is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Invenio; if not, write to the Free Software Foundation, Inc.,
+ * 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+* In applying this license, CERN does not
```
Comment (jirikuncar) [inline karma-dist-concatenated.conf.js]: missing new line before?

**Comment by jirikuncar** [comment]
- [ ] file with authors
- [ ] docs?

**Pull request merged**